src/main/java/com/zendesk/maxwell/producer/RabbitmqProducer.java

```diff
@@ -0,0 +1,57 @@
+package com.zendesk.maxwell.producer;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.zendesk.maxwell.MaxwellContext;
+import com.zendesk.maxwell.row.RowMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Created by hitesh on 30/12/16.
+ */
+public class RabbitmqProducer extends AbstractProducer {
+
+    static final Logger LOGGER = LoggerFactory.getLogger(RabbitmqProducer.class);
+    private static String exchangeName;
+    private Connection connection;
+    private Channel channel;
+    public RabbitmqProducer(MaxwellContext context) {
+        super(context);
+        this.exchangeName = context.getConfig().rabbitmqExchange;
+
+        ConnectionFactory factory = new ConnectionFactory();
+        factory.setHost(context.getConfig().rabbitmqHost);
+        try {
+            this.connection = factory.newConnection();
+            this.channel = connection.createChannel();
+            this.channel.exchangeDeclare(exchangeName, context.getConfig().rabbitmqExchangeType);
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (TimeoutException e) {
+            e.printStackTrace();
```

Comment:
rethrow these as runtime errors, I think.  If we can't connect to the channel we should crash here rather than in a separate thread.  Note that the producer error handling probably needs some tender loving care.